### PR TITLE
Blockuser feature

### PR DESCRIPTION
**Pull request by NidhiShah0511**
Aim - The following PR adds the ability for an Admin to block a user.

### CODE REVIEW

**Review by cesarguerrero1 — APPROVED** [review]
Complet and concise pull request!

File: interfaces/AdminDaoI.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ import User from "../models/users/User";
 
 export default interface AdminDaoI {
     findAllUsers(): Promise<User[]>;
+    blockUser(uid: string, user: User): Promise<User[]>;
```
Comment (SpatikaH) [inline interfaces/AdminDaoI.ts:5]: I guess `blockUser` should return a single user?

**Review by SpatikaH — APPROVED** [review]

**Comment by SpatikaH** [comment]
Looks good!

File: interfaces/AdminDaoI.ts (hunk 1 above)
Comment (NidhiShah0511) [reply interfaces/AdminDaoI.ts:5]: Thanks for pointing it out. Will change it in the next PR.

**Pull request merged**